sdk/eventhub/Azure.Messaging.EventHubs/src/Producer/EventHubBufferedProducerClientOptions.cs

```diff
@@ -32,7 +32,7 @@ public class EventHubBufferedProducerClientOptions
         private EventHubConnectionOptions _connectionOptions = new EventHubConnectionOptions();
 
         /// <summary>The set of options to govern retry behavior and try timeouts.</summary>
-        private EventHubsRetryOptions _retryOptions = new EventHubsRetryOptions();
+        private EventHubsRetryOptions _retryOptions = new EventHubBufferedProducerClientRetryOptions();
```

Comment:
Just curious - why not new up an `EventHubsRetyOptions` with the Properties set rather than add a new type here?

Reply:
Um.... yeah.   For some reason, I did not think of this at all.   Wow, now I feel silly.   I can just set these in the constructor.